<launch>
<node pkg="rosilo_denso_communication" type="rosilo_denso_communication_node" name="arm2">
<rosparam file="$(find rosilo_denso_communication)/cfg/smartarm_arm2.yaml" command="load"/>
</node>
</launch>
